diff --git a/pkg/usrlocalsharelima/usrlocalsharelima.go b/pkg/usrlocalsharelima/usrlocalsharelima.go
index df50871..b11153a 100644
--- a/pkg/usrlocalsharelima/usrlocalsharelima.go
+++ b/pkg/usrlocalsharelima/usrlocalsharelima.go
@@ -45,7 +45,7 @@ func Dir() (string, error) {
 		// - self:  /usr/local/bin/limactl
 		// - agent: /usr/local/share/lima/lima-guestagent.Linux-x86_64
 		// - dir:   /usr/local/share/lima
-		filepath.Join(selfDirDir, "share/lima/lima-guestagent.Linux-"+arch),
+		filepath.Join(selfDirDir, "share/codecomet/guestagent.Linux-"+arch),
 		// TODO: support custom path
 	}
 	for _, gaCandidate := range gaCandidates {
